frontend/app/routes/$lang+/_public+/apply+/$id+/review-information.tsx

```diff
@@ -174,44 +178,46 @@ export default function ReviewInformation() {
             </p>
           </DescriptionListItem>
           <DescriptionListItem term={t('apply:review-information.martial-title')}>
-            {userInfo.martialStatus}
+            {i18n.language === 'fr' ? maritalStatuses.find((x) => x.code === userInfo.martialStatus)?.nameFr : maritalStatuses.find((x) => x.code === userInfo.martialStatus)?.nameEn}
```

Comment:
```suggestion
            {maritalStatuses.find((ms) => ms.code === userInfo.martialStatus)?.[i18n.language === 'fr' ? 'nameFr' : 'nameEn']}
```

Reply:
For all these occurrences of `i18n.language === 'fr' ? 'nameFr' : 'nameEn'`, we can use `locale-utils` --> `getNameByLanguage(..)`

Reply:
changed to use `getNameByLanguage`